src/applications/disability-benefits/2346/actions/index.js

```diff
@@ -0,0 +1,37 @@
+import * as Sentry from '@sentry/browser';
+import { getVeteranInformation } from '../api';
+import {
+  FETCH_VETERAN_INFORMATION,
+  FETCH_VETERAN_INFORMATION_FAILURE,
+} from '../constants';
+
+export const fetchVeteranInformation = data => ({
+  type: FETCH_VETERAN_INFORMATION,
+  data,
+});
+
+export const fetchVeteranInformationFailure = error => ({
+  type: FETCH_VETERAN_INFORMATION_FAILURE,
+  error,
+});
+
+export const getVeteranInformationData = () => async dispatch => {
```

Comment:
This seems like a good opportunity for a unit test.

Reply:
We have a separate ticket for unit tests. Again we were asked by product owners and others to push this code to master. There will be smaller PR's coming that will address all requirements before actually deploying this application.